rockcraft/lifecycle.py

```diff
@@ -193,9 +193,23 @@ def _pack(
     emit.progress("Adding Pebble entrypoint")
 
     new_image.set_entrypoint()
-    if project.services:
-        new_image.set_pebble_services(
-            services=project.dict(exclude_none=True, by_alias=True)["services"],
+
+    services = (
+        project.dict(exclude_none=True, by_alias=True)["services"]
```

Comment:
(minor) `project.dict(exclude_none=True, by_alias=True).get("services", {})` might be clearer
or we could just call `project.dict()` once and use the result for both `services` and `checks`

Reply:
thanks for noticing. changed it